src/nvidia_rag/utils/vdb/milvus/milvus_vdb.py

```
@@ -540,6 +579,127 @@ def get_metadata_schema(
+                field_name="vector",
+                index_name="dense_index",
+                index_type="FLAT",
+                metric_type="L2",
+            )
+            client.create_collection(
+                collection_name=DEFAULT_DOCUMENT_INFO_COLLECTION,
+                schema=schema,
+                index_params=index_params,
+                consistency_level=CONSISTENCY_STRONG,
+            )
+            logger.info(f"Document info collection created at {self.vdb_endpoint}")
```

Ruff (0.14.5)

Logging statement uses f-string

(G004)